examples/CV7/CV7_example.cpp

```diff
@@ -76,110 +75,92 @@ void handleFilterEventSource(void*, const mip::Field& field, mip::Timestamp time
         exit_gracefully("ERROR: Could not ping the device!");
-    }
 
 
     //
-    //Idle the device (note: this is good to do during setup)
+    // Idle the device (note: this is good to do during setup)
     //
 
-    if( commands_base::setIdle(*device) != CmdResult::ACK_OK )
-    {
+    if (commands_base::setIdle(*device) != CmdResult::ACK_OK)
         exit_gracefully("ERROR: Could not set the device to idle!");
-    }
 
 
     //
-    //Load the device default settings (so the device is in a known state)
+    // Load the device default settings (so the device is in a known state)
     //
 
-    if( commands_3dm::defaultDeviceSettings(*device) != CmdResult::ACK_OK )
-    {
+    if (commands_3dm::defaultDeviceSettings(*device) != CmdResult::ACK_OK)
         exit_gracefully("ERROR: Could not load default device settings!");
-    }
 
 
     //
-    //Setup Sensor data format to 100 Hz
+    // Set up Sensor data format to 100 Hz
     //
 
     uint16_t sensor_base_rate;
 
-    //Note: Querying the device base rate is only one way to calculate the descriptor decimation.
-    //We could have also set it directly with information from the datasheet (shown in GNSS setup).
+    // Note: Querying the device base rate is only one way to calculate the descriptor decimation.
+    // We could have also set it directly with information from the datasheet (shown in GNSS setup).
 
-    if( commands_3dm::getBaseRate(*device, data_sensor::DESCRIPTOR_SET, &sensor_base_rate) != CmdResult::ACK_OK )
-    {
+    if (commands_3dm::getBaseRate(*device, data_sensor::DESCRIPTOR_SET, &sensor_base_rate) != CmdResult::ACK_OK)
         exit_gracefully("ERROR: Could not get sensor base rate format!");
-    }
 
     const uint16_t sensor_sample_rate = 100; // Hz
     const uint16_t sensor_decimation  = sensor_base_rate / sensor_sample_rate;
 
-    std::array<DescriptorRate, 4> sensor_descriptors = {
-        {
-            { data_shared::DATA_GPS_TIME, sensor_decimation },
-            { data_sensor::DATA_ACCEL_SCALED, sensor_decimation },
-            { data_sensor::DATA_GYRO_SCALED, sensor_decimation },
-            { data_sensor::DATA_MAG_SCALED, sensor_decimation },
-        }
-    };
+    std::array<DescriptorRate, 4> sensor_descriptors = { {
+        { data_shared::DATA_GPS_TIME,     sensor_decimation },
+        { data_sensor::DATA_ACCEL_SCALED, sensor_decimation },
+        { data_sensor::DATA_GYRO_SCALED,  sensor_decimation },
+        { data_sensor::DATA_MAG_SCALED,   sensor_decimation },
+    } };
 
-    if( commands_3dm::writeMessageFormat(*device, data_sensor::DESCRIPTOR_SET, sensor_descriptors.size(),
-                                         sensor_descriptors.data()) != CmdResult::ACK_OK )
-    {
+    if (commands_3dm::writeMessageFormat(*device, data_sensor::DESCRIPTOR_SET, sensor_descriptors.size(), sensor_descriptors.data()) !=
+        CmdResult::ACK_OK)
         exit_gracefully("ERROR: Could not set sensor message format!");
```

Comment:
I still don't like this line wrapping. In this case though, we don't need an explicit check for CmdResult::ACK_OK. Instead it could just be `if (!commands_3dm::writeMessageFormat(...))`